vanilla/vanilla
# Fix mass deletion of discussions causing response timeouts

closes https://github.com/vanilla/support/issues/974

Creating a job for batch deleting discussions.
### To Test
Create/Delete multiple discussions.(50 very large discussions, include images) You can increase the number of characters allowed on a discussion here
![Screen Shot 2020-03-20 at 3 13 11 PM](https://user-images.githubusercontent.com/31856281/77198463-64c1b100-6abd-11ea-93dd-e7b1345bbfcc.png)

Note the request delay (might be difficult to simulate on 10012 cluster with barely any traffic) but should still see a difference
